conversions.go

```diff
@@ -50,15 +52,23 @@ func convertInMessage(
 		}
 
 		o = &fuseops.LookUpInodeOp{
-			Parent:    fuseops.InodeID(inMsg.Header().Nodeid),
-			Name:      string(buf[:n-1]),
-			OpContext: fuseops.OpContext{Pid: inMsg.Header().Pid},
+			Parent: fuseops.InodeID(hdr.Nodeid),
+			Name:   string(buf[:n-1]),
+			OpContext: fuseops.OpContext{
```

Comment:
Instead of repeating this struct initialization over and over again, could you factor it out into a `opContext(hdr *fusekernel.InHeader) OpContext` helper function?